src/server.js

```diff
@@ -36,6 +36,11 @@ module.exports = function() {
 		.set("view engine", "html")
 		.set("views", path.join(__dirname, "..", "client"));
 
+	// Express will handle X-Forwarded-For header for us
+	if (Helper.config.reverseProxy) {
+		app.enable("trust proxy");
```

Comment:
` app.enable("trust proxy", true)` is not a thing.

> Calling app.set('foo', true) for a Boolean property is the same as calling app.enable('foo'). 

https://expressjs.com/en/4x/api.html#app.set